ui/app/AppLayouts/Chat/popups/community/CreateCommunityPopup.qml

```diff
@@ -95,6 +95,7 @@ StatusModal {
                 label: qsTr("Name your community")
                 charLimit: maxCommunityNameLength
                 input.placeholderText: qsTr("A catchy name")
+                text: ""
```

Comment:
Why this is needed ?

Reply:
Ah that was a test to fix the issue where the input would turn red on forcing the focus on it. It didn't work. I can remove it.